src/jira/connect.ts

```diff
@@ -21,13 +22,17 @@ export default (_: Request, res: Response): void => {
 		}
 	];
 
+	let apiMigrations: any = {gdpr: false}
+
+	if (await booleanFlag(BooleanFlags.USE_JWT_SIGNED_INSTALL_CALLBACKS, false)) {
+		apiMigrations = {gdpr: false, "signed-install": true}
+	}
```

Comment:
you could do it as just `const apiMigrations = {gdpr: false, "signed-install": await booleanFlag(BooleanFlags.USE_JWT_SIGNED_INSTALL_CALLBACKS, false)}`

Reply:
I don't want to change descriptor in any way if the feature flag is off. If I do it it might trigger a version update when we'll be performing migration.